packages/clarity-js/src/diagnostic/fraud.ts

```diff
@@ -10,6 +10,7 @@ export let data: FraudData;
 export function start(): void {
     history = [];
     metric.max(Metric.Automation, navigator.webdriver ? BooleanFlag.True : BooleanFlag.False);
+    metric.max(Metric.Iframed, window.top !== window.self ? BooleanFlag.True : BooleanFlag.False);
```

Comment:
We need to wrap this in try / catch or this will throw exception if the the current window doesn't have access to top most window.